torch/distributed/distributed_c10d.py

```diff
@@ -1614,7 +1614,7 @@ def all_gather_multigpu(
 
 def _object_to_tensor(obj, device):
     f = io.BytesIO()
-    _pickler(f).dump(obj)
+    torch.save(obj,f,pickle_module=pickle,pickle_protocol=4)
```

Comment:
Hmm, I think we need the pickler to be pluggable here still?

Reply:
I removed the pickle_module plugin